MetaLamp/nft-marketplace/test/Marketplace/Fixtures/Script.hs

```diff
@@ -19,11 +20,17 @@ marketplace =
   Marketplace.Marketplace {
     Marketplace.marketplaceOperator = walletPubKeyHash Fixtures.ownerWallet,
     Marketplace.marketplaceSaleFee = percentage,
-    Marketplace.marketplaceNFTFee = Lovelace 100000  -- 0.1 ADA
+    Marketplace.marketplaceNFTFee = marketplaceCreationFee
   }
 
+marketplaceCreationFee :: Ada
+marketplaceCreationFee = Lovelace 2_100_000 -- 2.1 ADA (should be gte then minAdaTxOut)
+
 percentage :: Percentage
-percentage = Percentage (5, 2)
+percentage = Percentage (7, 2)
+
+roundedPercentage :: Integer -> Integer
+roundedPercentage price = calculatePercentageRounded percentage price
```

Comment:
Probably we need to think about fees amount again, and change implementation, so that this is ensured everywhere:

1. If fee is a percentage, it is not less than `minAda`
2. If fee is fixed, but less than current `minAda`, it should be changed to `minAda`

What do you think?

Reply:
I already created the separate task for these changes.